libff/algebra/fields/tests/test_fields.cpp

```diff
@@ -18,7 +21,49 @@
 
 using namespace libff;
 
-#ifndef NDEBUG
+class FpnFieldsTest: public ::testing::Test {
+public:
+    FpnFieldsTest()
+    {
+        init_edwards_fields();
+        init_mnt4_fields();
+        init_mnt6_fields();
+        init_alt_bn128_fields();
+        init_bls12_381_fields();
+#ifdef CURVE_BN128 // BN128 has fancy dependencies so it may be disabled
```

Comment:
not sure if they're fancy, just external lol

Reply:
I didn't add this comment, I just moved it